types/ramda/test/filter-tests.ts

```diff
@@ -35,7 +35,9 @@ import * as R from "ramda";
 (() => {
     const coll = [{ type: "BUY" }, { type: "SELL" }, { type: "BUY" }];
     const isBuy = R.propEq("BUY", "type");
-    R.filter(isBuy, coll); // [{ type: 'BUY' }, { type: 'BUY' }]
+    // TODO
+    // R.filter(isBuy, coll); // => [{ type: 'BUY' }, { type: 'BUY' }]
```

Comment:
This I should not have left undo. A change in `types-ramda` needs to be reverted for this to pass again: https://github.com/ramda/types/pull/99
Until that is merged and released this MR should be considered blocked